src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -67,6 +69,16 @@ export class TaskListBuilder {
       )
     }
 
+    if (draft.isResponsePartiallyRejectedDueTo(RejectPartOfClaimOption.AMOUNT_TOO_HIGH)) {
```

Comment:
I do not see test case for this, please add so we are sure task list work properly.